### ISSUE TITLE
Group Pictures bug?

### ISSUE THREAD

**Issue by ivan-nginx** [theme-next/hexo-theme-next#713]
```
{% gp 5-2 %}
  ![](/images/docs/github.png)
  ![](/images/docs/next-default-scheme-linux.png)
  ![](/images/docs/tomorrow-night-blue.png)
  ![](/images/docs/tomorrow-night.png)
  ![](/images/docs/tomorrow-normal.png)
{% endgp %}
```
![image](https://user-images.githubusercontent.com/16944225/54605532-c7671380-4a49-11e9-9716-64e6ca2ff31b.png)

**Comment by ivan-nginx** [comment]
Is there any reason to continue support Post Gallery and / or Group Pictures features?

https://github.com/theme-next/hexo-theme-next/blob/8ee52feb2fa1f4649495aafeab61a9d1c4fb1f8d/layout/_macro/post.swig#L268-L281

As for me, this features too complicated with usage and no give good impact. For example, just with fancybox3 we can see something like gallery in right panel:

![image](https://user-images.githubusercontent.com/16944225/54619331-b11a8100-4a64-11e9-9d56-099982c9ccc7.png)

Also, we can try to find 3rd-party lib which will provide same feature and do it better.

Any comments or suggestions about this?

**Comment by stevenjoezhang** [comment]
> For example, just with fancybox3 we can see something like gallery in right panel:

@ivan-nginx It's not only created by fancybox. It will no longer exist if Post Gallery or Group Pictures feature is removed. See:
https://github.com/theme-next/hexo-theme-next/blob/8ee52feb2fa1f4649495aafeab61a9d1c4fb1f8d/source/js/src/utils.js#L8-L46

Would be better to keep these features

**Comment by stevenjoezhang** [comment]
@ivan-nginx I found the reason why `group-picture` won't display in post-detail:
https://github.com/theme-next/hexo-theme-next/blob/35180c26e9575eb63632107921d85b27ccc42af3/source/css/_common/components/tags/group-pictures.styl#L17-L22
It is broken by `float: none; `, which was added in: https://github.com/iissnan/hexo-theme-next/commit/7aa1095de2ab9890c1a6e8dfb9ab901119cc84ac#diff-b83e8966961c6a3d71d28e6cda1dd4feR17
I'm not sure if it's a bug or feature

**Comment by NashMiao** [comment]
This bug still exist... Maybe we need to fix it.

![image](https://user-images.githubusercontent.com/18191964/56938695-f9ce5b00-6b35-11e9-9852-343c0cda8db4.png)

https://theme-next.org/docs/tag-plugins/group-pictures

**Comment by stevenjoezhang** [comment]
Fixed in #929, #931
